src/game_renderer/MenuRenderer.cpp

```diff
@@ -0,0 +1,54 @@
+// Joseph Arhar
+
+#include "MenuRenderer.h"
+
+#include <imgui.h>
+#include <imgui_impl_glfw_gl3.h>
+#include <iostream>
+#include <string.h>
+
+#define TEXT_FIELD_LENGTH 256
+
+MenuRenderer::MenuRenderer() {}
+
+MenuRenderer::~MenuRenderer() {}
+
+void MenuRenderer::Render(GLFWwindow* window,
+                          std::shared_ptr<MenuState> menu_state) {
+  int width, height;
+  glfwGetFramebufferSize(window, &width, &height);
+  glViewport(0, 0, width, height);
+  glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+
+  static const ImGuiWindowFlags window_flags;
```

Comment:
I'm getting a compilation error from this
![screen shot 2017-02-15 at 9 32 07 pm](https://cloud.githubusercontent.com/assets/8975458/23008698/3e7e544c-f3c6-11e6-99d4-e7b627d8ed72.png)